src/fontAwesome.js

```diff
@@ -78,8 +79,25 @@ import userFriends from '@fortawesome/fontawesome-pro-regular/faUserFriends';
 import edit from '@fortawesome/fontawesome-pro-regular/faEdit';
 import expand from '@fortawesome/fontawesome-pro-light/faExpand';
 import lightQuestionCircle from '@fortawesome/fontawesome-pro-light/faQuestionCircle';
+import code from '@fortawesome/fontawesome-pro-regular/faCode';
+import lightbulb from '@fortawesome/fontawesome-pro-regular/faLightbulb';
+import users from '@fortawesome/fontawesome-pro-regular/faUsers';
+import language from '@fortawesome/fontawesome-pro-regular/faLanguage';
+
+// The following icons are having trouble being imported
+//
+// import fileSignature from '@fortawesome/fontawesome-pro-regular/faFileSignature';
+// import pencilPaintbrush from '@fortawesome/fontawesome-pro-regular/faPencilPaintbrush';
+// import voteYea from '@fortawesome/fontawesome-pro-regular/faVoteYea';
```

Comment:
@rej156 I'm having some trouble figuring out why these icons in particular are throwing errors when I try to import them. Uncomment them to see what I mean. Do you think you could take a look? Curious to hear what you think the issue might be.

Reply:
Just cooking haddock for dinner ATM, will check after bae

Reply:
No you won't! Enjoy your Sunday evening! I'm just wrapping up a few things because I'm heading out on vacation this week :) Can wait till Monday